rust/agama-lib/share/examples/profile.jsonnet

```
@@ -1,4 +1,10 @@
+// This is a Jsonnet file. Please, check https://jsonnet.org/ for more
+// information about the language.
```

OK, that's about the syntax.
As a user I'd also want to know the schema, so 

```suggestion
// information about the language.
// For the schema, see
// https://github.com/openSUSE/agama/blob/master/rust/agama-lib/share/profile.schema.json
```